### PR TITLE
feat: add afplay completion spec

### PR DESCRIPTION
**Pull request by Deep-Codes**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Feature: adds specs for `afplay`

### CODE REVIEW

File: dev/afplay.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      description: "print help",
+    },
+    {
+      name: ["--leaks"],
+      description: "Run leaks analysis.",
+    },
+    {
+      name: ["-t", "--time"],
+      description: `Play for \`TIME\` seconds
+      >0 and < duration of \`audio_file\`.`,
```
Comment (fedeci) [inline dev/afplay.ts:30]: I think we can add it to the description of the argument.

File: dev/afplay.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      args: {
+        name: "TIME",
+      },
+    },
+    {
+      name: ["-r", "--rate"],
+      description: `Play at playback \`RATE\`.
+      practical limits are about 0.4 (slower) to 3.0 (faster).`,
```
Comment (fedeci) [inline dev/afplay.ts:38]: Same here.

File: dev/afplay.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    template: "filepaths",
+    name: "play the song in the path",
+  },
+  options: [
+    {
+      name: ["-v", "--volume"],
+      description: `Set the volume for playback of the file
+      Apple does not define a value range for this, but it appears to accept
+      0=silent, 1=normal (default) and then up to 255=Very loud.
+      The scale is logarithmic and in addition to (not a replacement for) other volume control(s).`,
```
Comment (fedeci) [inline dev/afplay.ts:14]: And here too.

File: dev/afplay.ts (hunk 1 above)
Comment (Deep-Codes) [reply dev/afplay.ts:30]: I don't understand why 🤔 

if I add it in the `args`'s description I will get the autocomplete after i write the flag and the description would come for the flag's value.

Screenshot 2021-08-23 at 6 55 19 PM

Doesn't this seem more explanatory? 

Screenshot 2021-08-23 at 6 57 31 PM

Comment (fedeci) [reply dev/afplay.ts:30]: I don't mean everything, just the constraints part. Like
```js
  name: '-t',
  description: 'Play for TIME seconds,
  args: {
    name: 'TIME',
    description: 'A value greater than 0 but lower than the audio file length'
```

Comment (Deep-Codes) [reply dev/afplay.ts:30]: Thanks, sorry for the misunderstanding changing it.

**Comment by Deep-Codes** [comment]
@fedeci Done let me know 🙂

**Review by Nutlope — APPROVED** [review]

**Comment by Nutlope** [comment]
LGTM, thanks for the contribution @Deep-Codes!

**Pull request merged**